conf/projects/var-som-mx8m-nano/config.conf

```diff
@@ -31,6 +31,8 @@ TEST_TARGET_IP = "10.0.0.36"
 TEST_SERVER_IP = "10.0.0.13"
 TEST_TARGET = "simpleremote"
 
+do_image_wic[depends] += "imx-boot:do_deploy"
+
```

Comment:
I think we need this on other var-soms too right ?

Reply:
probably so